test/scss/_feature-targeting-test.scss

```diff
@@ -154,7 +154,7 @@
     @include mdc-elevation-overlay-common($query: $query);
     @include mdc-elevation(0, $query: $query);
     @include mdc-elevation-shadow(none, $query: $query);
-    @include mdc-elevation-overlay-parent($query: $query);
```

Comment:
This change seem unexpected.

Reply:
From what I can tell these mixins were removed. The error I was getting is that the mixin doesn't exist and I couldn't find it in a code search.